runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -251,6 +251,48 @@ J9::CodeGenerator::fastpathAcmpHelper(TR::Node *node, TR::TreeTop *tt, const boo
    // next treetop and then at the current one
    TR::Block* prevBlock = tt->getEnclosingBlock();
    TR::Block* targetBlock = prevBlock->splitPostGRA(tt->getNextTreeTop(), cfg, true, NULL);
+
+   // Move call tree to end of block, along with any stores of the result of the call, if they're
+   // not already at the end.  The call has to be split into its own block, but splitPostGRA above
+   // might have left register stores and stores into temporaries that must stay with the previous block
+   TR::TreeTop* prevBlockExit = prevBlock->getExit();
+   TR::TreeTop* afterTT = tt->getNextTreeTop();
```

Comment:
This is used as `iterator` mainly to walk through treetops. How about using `iterTT`?